inc/admin/ui/modules/advanced.php

```diff
@@ -26,7 +26,22 @@
 			'name'         => 'rocket_remove_query_strings_desc',
 			'description'  => __( '<strong>Note:</strong> By activating this option, you will improve the <code>Remove query strings from static resources</code> grade on GT Metrix.', 'rocket' ),
 		),
-
+		array(
+			'type'         => 'checkbox',
+			'label'        => __( 'Save CSS/JS generated by PHP to static files', 'rocket' ),
+			'name'         => 'cache_dynamic_resource',
+			'label_screen' => __( 'Save CSS/JS generated by PHP to static files', 'rocket' )
+		),
+		array(
+			'type'			=> 'helper_description',
+			'name'			=> 'rocket_cache_dynamic_resource_desc',
+			'description'  => __( 'This will save any CSS/JS content generated using PHP files as a static file instead.', 'rocket' )
+		),
+		array(
```

Comment:
Not sure it helps to write that. Each options shoul improve the page loading time.